pkg/status/status.go

```diff
@@ -176,8 +177,10 @@ func GetDCAStatus() (map[string]interface{}, error) {
 	apiCl, err := apiserver.GetAPIClient()
 	if err != nil {
 		stats["custommetrics"] = map[string]string{"Error": err.Error()}
+		stats["admission"] = map[string]string{"Error": err.Error()}
```

Comment:
maybe admission is too vague, what about `admissionwebhook`
 
```suggestion
		stats["admissionwebhook"] = map[string]string{"Error": err.Error()}
```